Zappy_IA_Report/DTC/Video/Tutorial2.py

```
@@ -0,0 +1,25 @@
+import pandas as pd
+
+df = pd.read_csv('500hits.csv', encoding='latin-1')
+print(df.head())
+df = df.drop(columns=['PLAYER', 'CS'])
+print(df.info())
+print(df.describe().round(3))
+X1 = df.iloc[:,0:13]
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)